test/conf/cassandra.yaml

```diff
@@ -8,6 +8,7 @@ memtable_allocation_type: offheap_objects
 commitlog_sync: batch
 commitlog_segment_size: 5MiB
 commitlog_directory: build/test/cassandra/commitlog
+commitlog_disk_access_mode : mmap
```

Comment:
mmap ? Aren't we going to test direct_io ?

Reply:
Default kept as it is and Direct-IO testing is enabled in "CommitLogStressTest.java" by adding a new entry. This entry will test "PeriodicCommitLogStressTest.java", "GroupCommitLogTest.java" and "BatchCommitLogTest.java". 

Let me know if this didn't solve the testing.

Reply:
commit log is mmap by default, so I don't think we need this , am I right ?

Reply:
> commit log is mmap by default, so I don't think we need this , am I right ?

I included for future usage if someone wants to change and test. Please suggest and will change as per the suggestion.

Reply:
`I included for future usage if someone wants to change and test.`
I recommend deleting it and leaving it for future revisionists to do the modifications